pkg/daemon/discover/discover.go

```diff
@@ -51,19 +51,28 @@ var (
 	cmName          string
 	cm              *v1.ConfigMap
 	udevEventPeriod = time.Duration(5) * time.Second
+	useCVInventory  bool
 )
 
-func Run(context *clusterd.Context, probeInterval time.Duration) error {
+func Run(context *clusterd.Context, probeInterval time.Duration, useCV bool) error {
 	if context == nil {
 		return fmt.Errorf("nil context")
 	}
 	logger.Infof("device discovery interval is %s", probeInterval.String())
+	logger.Infof("use ceph volume inventory is %t", useCV)
```

Comment:
```suggestion
	logger.Debugf("use ceph volume inventory is %t", useCV)
```

Reply:
I have changed both discovery command flags. It seems sensible to follow the same criteria for both. In any case, if there is any special interest in to keep as info the message for the interval, i can revert the change for this log line.